build-tools/android-toolchain/android-toolchain.projitems

```diff
@@ -30,11 +30,11 @@
       <HostOS>Darwin</HostOS>
       <DestDir>build-tools\$(XABuildToolsVersion)</DestDir>
     </AndroidSdkItem>
-    <AndroidSdkItem Include="platform-tools_r23-macosx.zip">
+    <AndroidSdkItem Include="platform-tools_r26.0.0-rc1-macosx.zip">
       <HostOS>Darwin</HostOS>
       <DestDir>platform-tools</DestDir>
     </AndroidSdkItem>
-    <AndroidSdkItem Include="tools_r24.4.1-macosx.zip">
+    <AndroidSdkItem Include="tools_r25.3.1-macosx.zip">
```

Comment:
My natural assumption on the Android SDK Tooling is that SDK Tools r25.3.1 and perhaps newer build-tools are required to support Android O API. At least we have to install "platform-O" things. But they are not available via the old Android SDK Manager (in the old SDK Tools). And that new SDK Tools version is only available as Beta version (Beta, Alpha or Canary). That means, to include Android O API support we will have to switch to Beta SDK Tools, even though our stable API is still 24 or 25.